### PR TITLE
[Fix]: The session wasn't working for other users

### PR DESCRIPTION
**Pull request by kprince28**
# Pull Request

## Description

- The session wasn't working as intended and that's why there was a problem with decryption

### CODE REVIEW

File: netbox_secrets/api/views.py [hunk 1]
```diff
@@ -202,7 +202,7 @@ def create(self, request):
         if master_key is None:
             return HttpResponseBadRequest(ERR_PRIVKEY_INVALID)
 
-        current_session_key = self.queryset.first()
+        current_session_key = self.get_queryset().first()
```
Comment (abhi1693) [inline netbox_secrets/api/views.py:205]: This should not be bypassed, instead we should look into why `self.queryset` does not contain the correct `QuerySet` defined in `get_queryset`

**Review by abhi1693 — CHANGES_REQUESTED** [review]

File: netbox_secrets/api/views.py (hunk 1 above)
Comment (dbrennand) [reply netbox_secrets/api/views.py:205]: Could it be due to: https://github.com/Onemind-Services-LLC/netbox-secrets/blob/dcedeba01920702c856e919065b23d3f1d0a9611/netbox_secrets/models/secrets.py#LL177C9-L177C18 ?

**Comment by abhi1693** [comment]
@kprince28 Since you were able to reproduce the issue, could you check if my changes fixes the issue?

**Comment by kprince28** [comment]
@abhi1693 The issue is fixed now. There is no problem when generating the season key for other users.

**Pull request merged**